docs/BR.md

```diff
@@ -1808,20 +1810,25 @@ If the Subordinate CA is not allowed to issue certificates with dNSNames, then t
 
 ### 7.1.6 Certificate policy object identifier
 
-#### 7.1.6.1. Reserved Certificate Policy Identifiers
 This section describes the content requirements for the Root CA, Subordinate CA, and Subscriber Certificates, as they relate to the identification of Certificate Policy.
 
-The following Certificate Policy identifiers are reserved for use by CAs as an optional means of asserting compliance with these Requirements as follows:
+#### 7.1.6.1. Reserved Certificate Policy Identifiers
+
+The following Certificate Policy identifiers are reserved for use by CAs as an optional means of asserting that a Certificate complies with these Requirements.
 
-{joint-iso-itu-t(2) international-organizations(23) ca-browser-forum(140) certificate-policies(1) baseline-requirements(2) domain-validated(1)} (2.23.140.1.2.1), if the Certificate complies with these Requirements but lacks Subject Identity Information that is verified in accordance with Section 3.2.2.1 or Section 3.2.3.
+When present within a Certificate that is not a Server Certificate (e.g. a Root CA Certificate or Subordinate CA Certificate, including Technically Constrained Subordinate CA Certificates and Subordinate CA Certificates that are Cross Certificates), the Issuing CA is asserting that the Certificate complies with these Baseline Requirements, and that the certificates issued by the Certificate shall also comply with these Baseline Requirements. Such CA Certificates MAY contain one or more of these reserved Certificate Policy identifiers, depending on the type of certificates that will be issued.
 
-If the Certificate asserts the policy identifier of 2.23.140.1.2.1, then it MUST NOT include organizationName, givenName, surname, streetAddress, localityName, stateOrProvinceName, or postalCode in the Subject field.
+When one of these Certificate Policy Identifiers is present within a Server Certificate, the Issuing CA is asserting that the Certificate complies with these Requirements, and that the Subject Identity Information has been verified as specified. Such Certificates MUST NOT contain more than one of these reserved Certificate Policy identifiers.
```

Comment:
Do we need to make "Reserved Certificate Policies Identifiers" a defined term and link to the publish list of value values? 
https://cabforum.org/object-registry/

Reply:
I'm loathe to link to the website from the Guidelines, both because of temporal mutability and because we often forget to update the website :)

Am I understanding that your concern is about ` 2.23.140.1.1` specifically? The EV certificate OID?

Reply:
I made an attempt to address the EV OID in https://github.com/sleevi/cabforum-docs/pull/21/commits/dfc0818cbff44bf484a54065ecf991fdbdc8798b

Reply:
I understand the issue with linking to the CABF site.  
Can we rely on the use of the term "these Certificate Policy Identifiers"  to mean exactly this list (or as modified from time to time), or is it best to include them here in the definition of "Reserved Certificate Policies Identifiers"? 
2.23.140.1.2.1
2.23.140.1.2.2
2.23.140.1.2.3
2.23.140.1.1

Reply:
I mean, this section is titled Reserved Certificate Policy Identifiers 😃 Maybe that's why I've got the disconnect on the concern.

I did upload https://github.com/sleevi/cabforum-docs/pull/21/commits/dfc0818cbff44bf484a54065ecf991fdbdc8798b to try to cover exactly those 4 OIDs :)

Reply:
Yes, I agree.  No issues from me.